Source/Core/Common/ArmCPUDetect.cpp

```diff
@@ -119,22 +191,20 @@ void CPUInfo::Detect()
 // Turn the CPU info into a string we can show
 std::string CPUInfo::Summarize()
 {
-  std::string sum;
-  if (num_cores == 1)
-    sum = fmt::format("{}, 1 core", cpu_string);
-  else
-    sum = fmt::format("{}, {} cores", cpu_string, num_cores);
+  std::vector<std::string> sum;
+  if (model_name.size())
+    sum.push_back(model_name);
+  if (cpu_id.size())
+    sum.push_back(cpu_id);
 
   if (bAES)
-    sum += ", AES";
+    sum.push_back("AES");
   if (bCRC32)
-    sum += ", CRC32";
+    sum.push_back("CRC32");
   if (bSHA1)
-    sum += ", SHA1";
+    sum.push_back("SHA1");
   if (bSHA2)
-    sum += ", SHA2";
-  if (CPU64bit)
-    sum += ", 64-bit";
+    sum.push_back("SHA2");
 
-  return sum;
+  return JoinStrings(sum, ",");
```

Comment:
Might be better to join by `", "` to keep the formatting consistent with older builds.

Reply:
Yea I have purposefully removed that because leading space makes it a tiny bit harder to parse the output (not that we do, but it could be nice to accumulate stats on analytics for example, to just do `feature_map[str.split()] += 1` or similar).

Reply:
Wouldn't this split the CPU type stats on analytics between old formatting and new formatting? Not sure if that's a concern.

Reply:
Yea, and I've changed formatting of first 2 elements as well: where it used to report something like
`AMD Ryzen Threadripper 3990X 64-Core Processor  (AuthenticAMD)`
it will now report
`AMD Ryzen Threadripper 3990X 64-Core Processor,F17M03S0`

on windows/arm64 it will report something like
`Microsoft SQ1 @ 3.0 GHz,ARMv8 (64-bit) Family 8 Model 805 Revision D0E`

This fixes: 
1) removes trailing space in the model name string (we were not following the x64 specification and stripping padding spaces)
2) reports the family/model/stepping information instead of the brand name, because FMS actually gives us useful information instead of the brand which is already in the model name (and the brand can be determined from the FMS ). FMS is also easily parsed and may be interesting to query server-side.
3) on windows/arm64 nothing was reported before for model/brand. now we'll get model info.

Breaking compat with the old format doesn't seem like a huge deal to me. In the future we might change it again (e.g. on arm64 - maybe just the raw MIDR would be better, similar to FMS).